documentation/modules/auxiliary/scanner/http/dolibarr_16_contact_dump.md

```diff
@@ -0,0 +1,215 @@
+## Vulnerable Application
+### Dolibarr 16 pre-auth contact database dump
+
+Dolibarr version 16 < 16.0.5 is vulnerable to a pre-authentication contact database dump.
+An unauthenticated attacker may retrieve a company’s entire customer file, prospects, suppliers,
+and potentially employee information if a contact file exists.
+Both public and private notes are also included in the dump.
+
+### Dolibarr GitHub Repository & Dolibarr setup with vulnerable docker image
+
+If you need to setup Dolibarr, this is the official GitHub Repository from Dolibarr or via docker with an unofficial but working image and docker-compose.
+``` 
+Official GitHub Repository:
+
+https://github.com/Dolibarr/dolibarr/tree/16.0.4 
+```
+
+```
+Unofficial image with docker-compose:
+
+docker pull tuxgasy/dolibarr:16.0.4
+
+Important: This image does not contain a database. So you need to link it with a database container.
+
+Use Docker Compose to integrate it with MariaDB (you can also use MySQL if you prefer).
+
+Create docker-compose.yml file as following:
+
+version: "3"
+
+services:
+    mariadb:
+        image: mariadb:latest
+        environment:
+            MARIADB_DATABASE: dolibarr
+            MARIADB_USER: dolibarr 
+            MARIADB_PASSWORD: dolibarr
+            MARIADB_RANDOM_ROOT_PASSWORD: 'yes'
+    web:
+        image: tuxgasy/dolibarr:16.0.4
+        environment:
+            DOLI_DB_HOST: mariadb
+            DOLI_DB_USER: dolibarr
+            DOLI_DB_PASSWORD: dolibarr
+            DOLI_DB_NAME: dolibarr
+            DOLI_URL_ROOT: 'http://0.0.0.0'
+            PHP_INI_DATE_TIMEZONE: 'Europe/Paris'
+        ports:
+            - "80:80"
+```
+
+Then run all services docker-compose up -d. Now, go to http://0.0.0.0 to access to the new Dolibarr installation.
```

Comment:
`http://0.0.0.0` is not a valid address. You probably mean to say go to the localhost?

Reply:
Well this is from the Docker hub repo documentation itself. And as a matter of fact http://0.0.0.0 will match http://127.0.0.1 although it is a weird way to put it I agree.